source/includes/steps-copy-cli.yaml

```
@@ -0,0 +1,113 @@
+title: Log In to MongoDB Cloud
+ref: copy-app-cli-log-in-to-mongodb-cloud
+content: |
+  To configure your app with {+cli-bin+}, you must log in to MongoDB Cloud using
+  an :atlas:`API key </configure-api-access/#programmatic-api-keys>` scoped to the organization or
+  project that contains the app.
+
+  .. code-block:: bash
+
+     realm-cli login --api-key="<MongoDB Cloud Public API Key>" --private-api-key="<MongoDB Cloud Private API Key>"
+
+---
+title: Pull the Latest Version of Your App
+ref: copy-app-cli-pull-the-latest-version-of-your-app
+content: |
+  You'll need a local copy of the configuration files for the {+app+} you're copying.
```

Per [MongoDB style guidelines](https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/), we should avoid future tense.

```suggestion
  You need a local copy of the configuration files for the {+app+} you're copying.
```